e2e/scaffolding/helpers.ts

```diff
@@ -470,26 +482,33 @@ export async function setEpochLength(keys: KeyringPair, epochLength: number): Pr
     assert.equal(
       actualEpochLength,
       TEST_EPOCH_LENGTH,
-      `should have set epoch length to TEST_EPOCH_LENGTH blocks, but it's ${actualEpochLength}`
+      `should have set epoch length to TEST_EPOCH_LENGTH blocks, but it's ${actualEpochLength}`,
     );
   } else {
     assert.fail('should return an EpochLengthUpdated event');
   }
 }
 
+export async function getNextRewardEraBlock(): Promise<number> {
```

Comment:
The changes for this file are the same as #2065, and this function will be used in the later changes.